actionview/lib/action_view/lookup_context.rb

```diff
@@ -53,28 +54,6 @@ module Accessors #:nodoc:
     register_detail(:variants) { [] }
     register_detail(:handlers) { Template::Handlers.extensions }
 
-    class DetailsKey #:nodoc:
```

Comment:
Seeing a decent amount of code (mostly vendored Rails; hard to exclude) that references this class directly: https://github.com/search?l=Ruby&q=detailskey&type=Code

Reply:
I've only been able to find vendored Rails copies referencing this.  It's been :nodoc:d forever. Do you think it's worthwhile to maintain the API?  I only moved the constant because of require order, but my plan is to move the resolver cache inside this class too (possibly those view path caches too)